### PR TITLE
Draft of Matching

### PR DESCRIPTION
**Pull request by laurenmbotha**
Added a temporary page to match a user to a driver based on date, display drivers that match that date

### CODE REVIEW

File: capstone/src/main/webapp/matching.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+
+function getDriver() {
+    return fetch('/register-driver').then(response => response.json());;
```
Comment (beneitzen) [inline capstone/src/main/webapp/matching.js]: Extra semi colon here.

File: capstone/src/main/webapp/matching.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+
+function findDateInDriver(date,drivers) {
```
Comment (beneitzen) [inline capstone/src/main/webapp/matching.js]: Put 1 space after a comma.

File: capstone/src/main/webapp/matching.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        let seats = available[i].seats;
+        let times = available[i].times;
+        let comment = document.createElement("p");
+        comment.innerText = first + " is avaiable during the " + times + " of this day. He has " + seats + " seats available in his car.";
+        document.getElementById("driver-container").appendChild(comment);
+    }
+}
+
+async function getMatch() {
+
```
Comment (beneitzen) [inline capstone/src/main/webapp/matching.js:35]: Remove blank line a the start of the function.

**Review by beneitzen — APPROVED** [review]
Looks good. Good structure with the functions and await + async.

Please fix the style comments before merging.

File: capstone/src/main/webapp/matching.js (hunk 1 above)
Comment (laurenmbotha) [reply capstone/src/main/webapp/matching.js]: removed semicolon

File: capstone/src/main/webapp/matching.js (hunk 2 above)
Comment (laurenmbotha) [reply capstone/src/main/webapp/matching.js]: Fixed

**Pull request merged**